pkg/apis/ignite/v1alpha1/conversion.go

```diff
@@ -84,3 +84,103 @@ func Convert_v1alpha1_OCIImageSource_To_ignite_OCIImageSource(in *OCIImageSource
 	out.ID, err = meta.ParseOCIContentID(contentRef)
 	return
 }
+
+// Convert_ignite_ImageSpec_To_v1alpha1_ImageSpec calls the autogenerated conversion function along with custom conversion logic
+func Convert_ignite_ImageSpec_To_v1alpha1_ImageSpec(in *ignite.ImageSpec, out *ImageSpec, s conversion.Scope) error {
+	if err := autoConvert_ignite_ImageSpec_To_v1alpha1_ImageSpec(in, out, s); err != nil {
+		return err
+	}
+
+	// Convert between the old and new OCI reference format
+	out.OCIClaim.Ref = in.OCIRef
+	out.OCIClaim.Type = ImageSourceTypeDocker
+
+	return nil
+}
+
+// Convert_v1alpha1_ImageSpec_To_ignite_ImageSpec calls the autogenerated conversion function along with custom conversion logic
+func Convert_v1alpha1_ImageSpec_To_ignite_ImageSpec(in *ImageSpec, out *ignite.ImageSpec, s conversion.Scope) error {
+	if err := autoConvert_v1alpha1_ImageSpec_To_ignite_ImageSpec(in, out, s); err != nil {
+		return err
+	}
+
+	// Convert between the old and new OCI reference format
+	out.OCIRef = in.OCIClaim.Ref
```

Comment:
I'd comment specifically here that `in.OCIClaim.Type` is ignored on purpose

Reply:
Added that comment to `Convert_v1alpha1_OCIClaim_To_ignite_OCI` in 9d6ab9e.